public/course_wizard.php

```diff
@@ -0,0 +1,261 @@
+                .ctb th{border-color:black;border-style:solid;border-width:1px;font-family:"Comic Sans MS", "Comic Sans";font-size:16px;font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
+                .ctb .tgPurp{background-color:#cbcefb;border-color:inherit;text-align:left;vertical-align:top}
+                .ctb .tgNorm{background-color:#ffffff;text-align:left;vertical-align:top}
+    </style>
+
+
+        <?php include "templates/header.php"?>
+        <h2>Class Creation Wizard!1!</h2>
+        <br>
+
+        <!-- Show current classes-->
+        <b>Courses</b>
+
+        <?php if (count($result_courses) > 0): ?>
+            <table class="ctb">
+                <thead>
+                    <tr>
+                        <th class="tgPurp">Course ID</th>
+                        <th class="tgPurp">Course Code</th>
+                        <th class="tgPurp">Course Name</th>
+                    </tr>
+        </thead>
+        <tbody>
+            <?php foreach($result_courses as $row):;?>
+            <tr>
+                <td><?php echo ($row->id);?></td>
+                <td><?php echo ($row->course_code);?></td>
+                <td><?php echo ($row->course_name);?></td>
+            </tr>
+            <?php endforeach;?>
+            <tr>
+                <td></td>
+                <form method="post">
+                <td><input type="text" value="Course Code" name="course_code" id="course_code"></td>
+                <td><input type="text" value="Course Name" name="course_name" id="course_name"></td>
+                <td><input type="submit" name="submitCourse" value="Add"></td>
+            </tr>
+        </tbody>
+        </table>
+        <?php else:?>
+            <b>No courses found. Add a course?</b>
+            <form method="post">
+            <input type="text" value="Course Code" name="course_code" id="course_code">
+            <input type="text" value="Course Name" name="course_name" id="course_name">
+            <input type="submit" name="submitCourse" value="Add">
+        <?php endif;?>
+
+<!--Offerings-->
+
+        <br><br>
+        <b>Offerings</b>
+        <br>
+        <?php if (count($result_offerings) > 0): ?>
+            <table class="ctb">
+                <thead>
+                    <tr>
+                        <th class="tgPurp">Offering ID</th>
+                        <th class="tgPurp">Course ID</th>
+                        <th class="tgPurp">Course Name</th>
+                        <th class="tgPurp">Offering Code</th>
+                        <th class="tgPurp">Offering Name</th>
+                    </tr>
+        </thead>
+        <tbody>
+            <?php foreach($result_offerings as $row):;?>
+            <tr>
+                <td class="tgNorm"><?php echo ($row->id);?></td>
+                <td class="tgNorm"><?php echo ($row->course_id);?></td>
+                <td class="tgNorm"><?php echo ($result_courses[$row->course_id-1]->course_name);?></td>
```

Comment:
this line is likely to cause issues if courses get deleted, maybe not the best idea

Reply:
My immediate thought was to recommend doing a sequential search of $result_courses based on the ID, but then I realised that such a solution might not be scalable because the number of courses may get large, in which case it might be more prudent to not load the entire list of courses. Anyway, to remedy this problem, have you considered using the `belongs_to` association? That way you can get away with doing `$row->course->course_name` instead

Reply:
Thank you for this suggestion too, much better and cleaner. Added belongs_to association to respective classes and used those instead!